functions-and-scope.js

```diff
@@ -42,6 +59,18 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6.642857142857143
 
+let totalGrades = 0;
+let totalAmount = 0;
+
+for (let i = 0; i < grades.length; i++) {
+    totalGrades = totalGrades + grades[i];
```

Comment:
Dit kun je verkorten naar:
```suggestion
    totalGrades += grades[i];
```